addons/l10n_dk/data/template/account.tax-dk.csv

```diff
@@ -1,111 +1,111 @@
+"tax410","410","25% G INC","VAT 25% included, goods","25% included","25.0","percent","purchase","True","","base","invoice","","","","Købsmoms 25% indeholdt, varer"
 "","","","","","","","","","","tax","invoice","a8740","+KM","",""
 "","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","tax","refund","a8740","-KM","",""
-"tax420","420","VAT 25%, services","","25%","25.0","percent","purchase","","","base","invoice","","","","Købsmoms 25%, ydelser"
+"tax420","420","25% S","VAT 25%, services","25%","25.0","percent","purchase","","","base","invoice","","","","Købsmoms 25%, ydelser"
 "","","","","","","","","","","tax","invoice","a8740","+KM","",""
 "","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","tax","refund","a8740","-KM","",""
-"tax425","420","VAT 25% included, services","","25%","25.0","percent","purchase","True","","base","invoice","","","","Købsmoms 25% indeholdt, ydelser"
+"tax425","420","25% S INC","VAT 25% included, services","25%","25.0","percent","purchase","True","","base","invoice","","","","Købsmoms 25% indeholdt, ydelser"
 "","","","","","","","","","","tax","invoice","a8740","+KM","",""
 "","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","tax","refund","a8740","-KM","",""
-"tax430","430","Purchase reverse charge","","","25.0","percent","purchase","","","base","invoice","","","","Køb omvendt betalingspligt"
+"tax430","430","25% R C","Purchase reverse charge","","25.0","percent","purchase","","","base","invoice","","","","Køb omvendt betalingspligt"
 "","","","","","","","","","","tax","invoice","a8740","+KM","",""
 "","","","","","","","","","","tax","invoice","a8725","-UM","-100",""
 "","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","tax","refund","a8740","-UM","",""
 "","","","","","","","","","","tax","refund","a8725","+KM","-100",""
-"tax450","450","Restaurant VAT 6.25%, purchase VAT","","6,25%","25.0","percent","purchase","","","base","invoice","","","","Restaurationsmoms 6,25%, købsmoms"
+"tax450","450","6,25% R1","Restaurant VAT 6.25%, purchase VAT","6,25%","6.25","percent","purchase","","","base","invoice","","","","Restaurationsmoms 6,25%, købsmoms"
 "","","","","","","","","","","tax","invoice","a8740","+KM","25",""
 "","","","","","","","","","","tax","invoice","","","75",""
 "","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","tax","refund","a8740","+KM","25",""
 "","","","","","","","","","","tax","refund","","","75",""
-"tax460","450","Restaurant VAT 6.25%, purchase VAT","","6,25%","25.0","percent","purchase","True","","base","invoice","","","","Restaurationsmoms 6,25%, købsmoms"
+"tax460","450","6,25% R2","Restaurant VAT 6.25%, purchase VAT","6,25%","6.25","percent","purchase","True","","base","invoice","","","","Restaurationsmoms 6,25%, købsmoms"

```

Comment:
This tax is very similar to the previous one, and they share the same description. Do we really need both ? And what is this "6, 25%" thingy ? :o

Reply:
Since one of them is price included i have change a bit the name, description and translation

Reply:
@oco-odoo Just to answer for the 6.25% because I've worked a bit on it.
The answer is that in DK, you pay 25% of VAT and for restaurant, you can deduce 25% of it.
25% * 25% = 6.25%

I don't know why the partner put 6.25% when they introduced it. It might be because it's what they're used to see in other software. :man_shrugging: